apollo-runtime/src/main/java/com/apollostack/android/GraphqlService.java

```diff
@@ -0,0 +1,14 @@
+package com.apollostack.android;
+
+import android.database.Observable;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/** A {@link retrofit2.Retrofit} service for executing GraphQL requests. */
+public interface GraphqlService {
```

Comment:
Yeah my idea was that you'd extend this interface, but now that we're talking about it, it doens't make a lot of sense. Maybe this should be generated by the compiler as well, I'm not sure. Or maybe the user should implement it themselves and we don't provide it at all.
What do you think?